src/searchee.ts

```diff
@@ -87,10 +88,17 @@ export async function createSearcheeFromPath(
 		(runningTotal, file) => runningTotal + file.length,
 		0,
 	);
+	const seasonMatch =
+		basename(filepath).length < 12
+			? basename(filepath).match(SONARR_SUBFOLDERS_REGEX)
+			: undefined;
+
 	return resultOf({
 		files: files,
 		path: filepath,
-		name: basename(filepath),
+		name: seasonMatch
+			? `${basename(dirname(filepath))} S${seasonMatch?.groups?.season}`
```

Comment:
This will be `Series Title (YEAR) {tvdb-xxxx} SXX` right?

Reply:
If IDs were included (I don't include IDs) then I believe they would also be included in the searchee.name, yes. Should this be filtered? 

I don't think that would cause issues with arr lookups, but it may with non-ID based searches....

interesting thought.